unified-codes/apps/web/src/components/containers/detail/DetailPropertyListItem.tsx

```diff
@@ -64,7 +64,7 @@ const DetailPropertyListItem: DetailPropertyListItem = ({ description, propertie
     const childProperties = properties?.map((property: IProperty) => {
       const { type, value, properties } = property;
       const description = `${type}: ${value}`;
-      return <DetailPropertyListItem description={description} properties={properties} />;
+      return <DetailPropertyListItem key={description} description={description} properties={properties} />;
```

Comment:
This one is a bit trickier, but it feels a bit yucky to use a UI string for logic? Since properties won't be duplicated for the same parent, suggest just using `key={type]`.

Reply:
```suggestion
      return <DetailPropertyListItem key={type} description={description} properties={properties} />;
```

Reply:
Cool yeah, type is probably better - I don't think we have any duplicates at the moment (can revisit if it does happen) so should work nicely.